### ISSUE TITLE
Cannot load pages using serve due to violation of security directive

### ISSUE THREAD

**Issue by mandibuswell** [ServiceInnovationLab/FamilyServiceDirectorySearchinator#127]
I have followed the process.
Forked
Cloned
npm install
npm build
installed serve
run serve

So I have the serve server running on port 5000 but when I got to the browser either on chrome or google the page will not load due to a violation as follows

Refused to load the stylesheet 

Content Security Policy: The page’s settings blocked the loading of a resource at http://xxxxxx:5000/FamilyServiceDirectorySearchinator/static/css/main.525ea56c.css (“style-src https://api.addressfinder.io https://fonts.googleapis.com https://unpkg.com https://serviceinnovationlab.github.io 'unsafe-inline'”).
SyntaxError: expected expression, got '<'

the service runs in development fine. just fails for a production build.

**Comment by Br3nda** [comment]
@mandibuswell Does it work better if you run `npm start` ?

**Comment by mandibuswell** [comment]
good question Brenda. I was able to run it with the development server  so good thought cause that would mean a problem in the build process, so I tried it again and looking at the console with firebug I do still get the same error but in this case it ignores it

Content Security Policy: Couldn’t parse invalid host 'unsafe-inline'
Content Security Policy: Ignoring duplicate source 'unsafe-inline'
Content Security Policy: Couldn’t parse invalid host 'unsafe-inline'

I also see an additional one from bundle.js
Content Security Policy: The page’s settings blocked the loading of a resource at ws://xxxxx:3000/sockjs-node/263/gr3ls0po/websocket (“connect-src http://xxxxx:3000 ws://localhost:3000 https://catalogue.data.govt.nz https://api.addressfinder.io”).

I've tried to find a way to ignore the CSP unsafe-inline  for serve but could not find anything.
